lib/import-from-tiled.js

```diff
@@ -0,0 +1,135 @@
+"use strict";
+
+/** <p>Import an orthogonal tilemap from
+ * <a href="http://www.mapeditor.org/" target="_blank">Tiled: Map Editor</a> and draw on the screen.</p>
+ * <h5>To Do</h5>
+ * <ul>
+ * <li><b>Image Layer</b></li>
+ * <li><b>Ellipse and Polygon Objects</b></li>
+ * </ul>
+ * @module Import From Tiled
+ */
+
+/** @function ImportTilemap
+ * @param {Object} file JSON file exported from Tiled. This should be required in a scene enter
+ * script and passed to the function.
+ * @param {EntityPool} entities EntityPool from game.entities
+ */
+function ImportTilemap(file, entities) { // eslint-disable-line no-unused-vars
+
+	var tile, image, image_index, tile_pos, cols, tileset, tileset_index = 0;
+	var layer, object, entity, object_properties = {};
+
+	for (var i = 0; i < file.layers.length; i++) {
+		layer = file.layers[i];
+
+		// Render Tile Layers
+		if (layer.type == "tilelayer") {
+			for (j = 0; j < layer.data.length; j++) {
+
+				// Select tileset index based on firstgid and image index
+				// Set image index minus firstgid for positioning math
+				for (var k = 0; k < file.tilesets.length; k++) {
+					if (layer.data[j] >= file.tilesets[k].firstgid) {
+						tileset_index = k;
+					}
+				}
+				cols = file.tilesets[tileset_index].imagewidth / file.tilesets[tileset_index].tilewidth;
+				image_index = layer.data[j] - file.tilesets[tileset_index].firstgid;
+				tileset = file.tilesets[tileset_index];
+
+				if (image_index >= 0) {
+
+					// Create tile and get properties
+					tile = entities.create();
+					entities.set(tile, "name", "tile");
+					entities.set(tile, "tile", true);
+					entities.set(tile, "image", { "name": "" });
```

Comment:
It's weird to `set` a blank `image`, then `get` it and modify it later. Just set it once.